VCU/Phantom/tasks/source/task_throttle.c

```diff
@@ -75,164 +70,136 @@ extern volatile uint8_t initializationOccured;
  * @return                  - None
  * @Note                    - None
  ***********************************************************/
-void vThrottleTask(void *pvParameters){
+void vThrottleTask(void *pvParameters)
+{
 
-
-    TickType_t xLastWakeTime;          // will hold the timestamp at which the task was last unblocked
-    //const TickType_t xFrequency = 10 ; // task frequency in ms
+    TickType_t xLastWakeTime;         // will hold the timestamp at which the task was last unblocked
+    const TickType_t xFrequency = 5; // task frequency in ms - need to come up with this number better
 
     // Initialize the xLastWakeTime variable with the current time;
     xLastWakeTime = xTaskGetTickCount();
 
-    while(true)
-    {
-        if(initializationOccured){
-            // Wait for the next cycle
-            vTaskDelayUntil(&xLastWakeTime, THROTTLE_TASK_PERIOD_MS);
-
-            // for timing:
-            gioSetBit(hetPORT1, 5, 1);
-
-            // read APPS signals
-            if (TASK_PRINT) {UARTSend(PC_UART, "THROTTLE CONTROL\r\n");}
-    //        UARTSend(scilinREG, xTaskGetTickCount());
-
-            // how was this i from 0 to 10 selected?
-    //        for(i=0; i<10; i++)
-    //        {
-    //            adcStartConversion(adcREG1, adcGROUP1);
-    //            while(!adcIsConversionComplete(adcREG1, adcGROUP1));
-    //            adcGetData(adcREG1, 1U, FP_data_ptr);
-    //            FP_sensor_1_sum += (unsigned int)FP_data[0].value;
-    //            FP_sensor_2_sum += (unsigned int)FP_data[1].value;
-    //            BSE_sensor_sum  += (unsigned int)FP_data[2].value;
-    //        }
-
-            adcStartConversion(adcREG1, adcGROUP1);
-            while(!adcIsConversionComplete(adcREG1, adcGROUP1));
-            adcGetData(adcREG1, 1U, FP_data_ptr);
-            BSE_sensor_sum = (unsigned int)FP_data[0].value; // ADC pin 19?
-            FP_sensor_1_sum = (unsigned int)FP_data[1].value; // ADC pin 20?
-            FP_sensor_2_sum = (unsigned int)FP_data[2].value; // ADC pin 21?
-
-
-            // check for short to GND/5V on sensor 1
-            // thresholds
-
-            // check for short to GND/3V3 on sensor 2
-            // thresholds
-
-            // check for short to GND/5V on BSE
-            if (BSE_sensor_sum < BSE_MIN_VALUE)
-            {
-                // if it's less than 0.5V, then assume shorted to GND as this is not normal range
-                VCUDataPtr->DigitalVal.BSE_FAULT = 1;
+    // timer started is false to begin with
+    APPS1_range_fault_timer_started = false;
+    APPS2_range_fault_timer_started = false;
+    BSE_range_fault_timer_started   = false;
+    FP_diff_fault_timer_started     = false;
```

Comment:
These statements do not need to be here as they are already initialized to false during their declarations.

Reply:
okay I removed them.